### ISSUE TITLE
NVDA 2021 and Baum Vario Pro

### ISSUE THREAD

**Issue by FHKopp** [nvaccess/nvda#12817]
### Steps to reproduce:
I installed the update to NVDA 2021.01 and installed the driver for my Baum Vario Pro 80 too. The cursorrouting is not working so well as before. If i take the cursorrouting in a Textdocument it works (but it needs a doubleclick to get the cursor to the wished place, before a single click was enough). In other apps like Browser it doesn't work. It is not possible to get the cursor to the clicked position. I've contact the Programmer of the driver and he told me, that it is an issue of NVDA. Is there a solution to fix it? I didn't find anything about it. Additional there is a second problem, but i don't know if it is a problem of NVDA or of Code Factory. I installed the add-on of Code Factory for eloquence voices and if the checkbox to speak "capital" for capital letters several letters additional to the capital letter will be spoken (but they don't exist in the word).

#### NVDA installed/portable/running from source:
Installed NVDA Version 
#### NVDA version:
NVDA 2021
#### Windows version:
Windows 10 professional
#### Name and version of other software in use when reproducing the issue:
Office 2016, Notepad, Google Chrome and Firefox in current versions
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
In older versions of NVDA the Problem doesm didn't exist
#### If add-ons are disabled, is your problem still occurring?
yes

**Comment by Adriani90** [comment]
cc: @leonardder, @michaelDCurran

**Comment by feerrenrut** [comment]
Please ask the developer of the driver to raise an issue with details of the problem.

Please open individual issues for each problem and fill in the template in future.

Note that our issue tracker (GitHub issues) is intended for a technical audience who can investigate and provide detailed technical descriptions, if you require support for NVDA please consider support offered by NV Access: https://www.nvaccess.org/product/nvda-telephone-support/

**Comment by FHKopp** [comment]
i have contacted the NVDA Support and i found a support for Code Factory.
